cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -1544,6 +1544,29 @@ spec:
                   description: Network related configuration
                   nullable: true
                   properties:
+                    connections:
+                      description: Settings for network connections such as compression and encryption across the wire.
+                      nullable: true
+                      properties:
+                        compression:
+                          description: Compression settings for msgr2.
+                          nullable: true
+                          properties:
+                            enabled:
+                              description: Whether to compress the data in transit across the wire. The default is not set. Requires Ceph Quincy (v17) or newer.
+                              nullable: true
+                              type: boolean
+                          type: object
+                        encryption:
+                          description: Encryption settings for msgr2.
```

Comment:
Should we leave `msgr2` out of the description here? I wonder if it might be good to hide that detail from users so they don't feel the need to dig into Ceph internals.

Reply:
I think we should still mention msgr2 in the docs in case they want to dig in, but agreed no need to mention it here in the crd description